README.md

```diff
@@ -50,8 +50,8 @@ a pull request is a great way to become part of the team.
    1. Change the email & password values to something of your liking
    1. Run the script: `rails runner script/admin_tasks/create_user.rb`
    1. Run the app: `rails server`
-   1. Open the app in your browser: `http://localhost:3000`
-   1. Make sure you can log in with your local user
+   1. Open the app in your browser: `http://localhost:3000/2021/sign_ups/sign_in`
+   1. Sign In with username: `admin@localhost.local` and password: `foofighters`.
```

Comment:
Hi Michael, I like the suggestion to have a generic email in `create_user.rb`.

I'd prefer not to change the `sign_in` URL here every year. Hopefully developers can find an click on a "Sign In" link. :) Also, given the instructions above ("Change the [login] to something of your liking"), I'd suggest something like:

```suggestion
   1. Open the app in your browser: `http://localhost:3000`
   1. Sign In with the username and password that you configured above, in `create_user.rb`
```

Reply:
The only issue I have with the step: "Sign In with the username and password that you configured above, in `create_user.rb`" is that we may be introducing the idea that this file should be changed. This creates the potential for a developer to accidentally commit their changes. What we could do is write an example that someone could copy if we want to suggest that create_user.rb be changed.

Reply:
Good point about not committing changes to `create_user`. Maybe we should delete these two lines:

```diff
-   1. Create a new admin user for yourself by editing `/script/admin_tasks/create_user.rb`
-   1. Change the email & password values to something of your liking
```